xbmc/utils/CPUInfo.cpp

```diff
@@ -11,6 +11,7 @@
 #include "CPUInfo.h"
 #include "utils/log.h"
 #include "utils/Temperature.h"
+#include <utils/SysfsUtils.h>
```

Comment:
change to `#include "utils/SysfsUtils.h"` and move it up one line as  the includes should be sorted (https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#6-headers)